cloud-agent/service/server/src/main/scala/org/hyperledger/identus/presentproof/controller/http/PresentationStatusPage.scala

```diff
@@ -46,18 +46,61 @@ object PresentationStatusPage {
         )
     object next
         extends Annotation[String](
-          description = "URL of the next page (if available)",
-          example = ""
+          description =
+            "An optional string field containing the URL of the next page of results. If the API response does not contain any more pages, this field should be set to None.",
+          example = "/present-proof/presentations?offset=20&limit=10"
         )
     object previous
         extends Annotation[String](
-          description = "URL of the previous page (if available)",
-          example = ""
+          description =
+            "An optional string field containing the URL of the previous page of results. If the API response is the first page of results, this field should be set to None.",
+          example = "/present-proof/presentations?offset=0&limit=10"
         )
     object contents
         extends Annotation[Seq[PresentationStatus]](
           description = "A sequence of Presentation objects.",
-          example = Seq.empty
+          example = Seq(
```

Comment:
Have you checked that this `Seq` is rendered correctly in the API doc? As far as I can see in the YAML spec file above, it looks weird... 🤔

Reply:
yes I points that out but is a more generic problem.

Its already on the OpenAPI on other stuff like the class `EntityResponse`.

https://staging-docs.atalaprism.io/agent-api/#tag/Identity-and-Access-Management/operation/getAllEntities

![2024-06-03-15_55_07](https://github.com/hyperledger/identus-cloud-agent/assets/4622245/59503de7-d994-493b-9fd4-0a59dd8b3998)